guides/ci.md

```diff
@@ -0,0 +1,70 @@
+# Continuous Integration and automated checks
+
+This guide explains what runs locally versus in CI, how visual testing works, and how to act on failures.
+
+## Overview
+
+- Local: `yarn test` (or `npm run test`) via `dotrun test` runs spelling, formatting/linting, Parker metrics, and Jest tests. Some tasks (e.g., Percy) are CI-only.
+- CI (GitHub Actions): Percy visual tests run on PRs that meet selectivity rules or when labeled. See [Percy workflow](/guides/percy-workflow.md).
+- Legacy deployment jobs run in Jenkins for PS5 (high-level only; see [Deployment](/guides/deployment.md)).
+
+## Local checks
+
+Typical commands:
+
+```
+dotrun test
+yarn test
```

Comment:
only needs `yarn test`, with maybe a mention that `dotrun <command>` runs a command inside the dotrun image for better isolation. it's more important that maintainers run `npx prettier -c -w .` to apply formatting fixes.